src/hooks/useDragAndDropSubThought.ts

```diff
@@ -118,13 +118,18 @@ const drop = (props: DroppableSubthoughts, monitor: DropTargetMonitor) => {
   if (equalPath(thoughtsFrom, props.simplePath)) return
 
   // cannot move root or em context or target is divider
-  if (isDivider(thoughtTo.value) || (isRootOrEM && !sameContext)) {
+  if (isDivider(thoughtTo?.value) || (isRootOrEM && !sameContext)) {
     store.dispatch(
       error({ value: `Cannot move the ${isEM(thoughtsFrom) ? 'em' : 'home'} context to another context.` }),
     )
     return
   }
 
+  if (!thoughtTo) {
+    console.warn(`Cannot drop ${thoughtFrom} on itself. Aborting drop.`)
+    return
+  }
```

Comment:
This `console.warn` is not accurate, as dropping a thought on itself is already detected and short circuited using the `equalPath` check above. Please update the message or remove the `console.warn` completely, combining it with the first return statement.